packages/example-app-base/src/utilities/customizations.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import { ICustomizations } from 'office-ui-fabric-react/lib/Utilities';
+
```

Comment:
IIRC, I think the Fabric internal standard is to do package-level imports of main OUFR library since 6.0. David even when to change a whole bunch of internal imports to be package-based ...

Reply:
I'm not sure I follow. I'm seeing 65 instances of localized OUFR imports in `example-app-base` alone.

Reply:
I concur with @cliffkoh on this one - having explicit "/lib" locks people into "ES6." This affects webpack and jest... so, it's nice to move onto the new and just start deprecating the old.